src/Core/ServerSettings.cpp

```diff
@@ -562,6 +562,24 @@ namespace DB
     <max_table_num_to_warn>400</max_table_num_to_warn>
     ```
     )", 0) \
+    DECLARE(UInt64, max_pending_mutations_to_warn, 5000lu, R"(
+    If the number of pending mutations exceeds the specified value, clickhouse server will add warning messages to `system.warnings` table.
+
+    **Example**
+
+    ``` xml
+    <max_pending_mutations_to_warn>400</max_pending_mutations_to_warn>
+    ```
+    )", 0) \
+    DECLARE(UInt64, max_stuck_mutations_to_warn, 5000lu, R"(
```

Comment:
5000 is probably too much, it's even higher than `number_of_mutations_to_delay` and `number_of_mutations_to_throw`

Reply:
Bumped it down to 500, looks like the [default](https://github.com/ClickHouse/ClickHouse/blob/ad51570203338321c884fc13d2255438f6f12418/src/Storages/MergeTree/MergeTreeSettings.cpp#L112) for max_mutations_to throw 1000.